src/pages/nutrition/productCard.jsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Zoom from "react-medium-image-zoom";
+import Button from "@mui/material/Button";
+import  Typography  from '@mui/material/Typography'
+import offService from '../../off'
+import { useEffect } from "react";
+
+export default function ProductNutriments({}) {
+
+  const { getNutritionToFillUrl } = offService
+
+  const [ page, setPage ] = React.useState(1)
+  const [ index, setIndex ] = React.useState(0)
+  const [ products, setProducts ] = React.useState([])
+
+  useEffect(() => {
+    console.log('fetched')
+    const productListUrl = getNutritionToFillUrl({page})
+  fetch(productListUrl)
+    .then(res => res.json())
+    .then(data => setProducts(data.products))
```

Comment:
maybe set the `index` to 0, or keep the previous products with `setPoducts(prevProducts => [...prevProducts, ...data.products])`